crates/mcp-server/src/tools.rs

```diff
@@ -16,6 +16,9 @@ use crate::components::{
     handle_load_component, handle_unload_component,
 };
 
+/// The list of components that Wassette knows about
+const COMPONENT_LIST: &'static str = include_str!("../../../component-registry.json");
+
```

Comment:
Can you make this a CLI flag? 

Like `wassette --component-list <PATH>`

Reply:
Discussed offline - we agreed to go with what this is and can add once we have actual registry support.